feature/experimental/isis/otg_tests/internal/session/session.go

```diff
@@ -227,16 +226,11 @@ func (s *TestSession) ConfigISIS(ocFn func(*oc.NetworkInstance_Protocol_Isis)) {
 
 // PushAndStart calls PushDUT and PushAndStartATE to send config to both
 // devices.
-func (s *TestSession) PushAndStart(t *testing.T) error {
+func (s *TestSession) PushAndStart(t testing.TB) error {
 	t.Helper()
 	if err := s.PushDUT(context.Background()); err != nil {
 		return err
 	}
-	if *deviations.ExplicitInterfaceInDefaultVRF {
```

Comment:
Line 235-239 are expected to delete?

Reply:
same as https://github.com/openconfig/featureprofiles/pull/939#discussion_r1062980594

Reply:
SG